pkg/operator/ceph/csi/spec.go

```diff
@@ -294,6 +296,11 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		tp.EnableOMAPGenerator = true
 	}
 
+	// if k8s >= v1.20 enable oidc token projection.
```

Comment:
Can we remove the `if k8s >= v1.20` this looks like code than human-readable comment :)

Reply:
done, modified the comment.